streams/piping/abort.any.js

```diff
@@ -53,29 +53,32 @@ promise_test(t => {
       });
 }, 'an aborted signal should cause the writable stream to reject with an AbortError');
 
-promise_test(() => {
-  let error;
-  const rs = recordingReadableStream(errorOnPull, hwm0);
-  const ws = new WritableStream();
-  const abortController = new AbortController();
-  const signal = abortController.signal;
-  abortController.abort();
-  return rs.pipeTo(ws, { signal })
-      .catch(e => {
-        error = e;
-      })
-      .then(() => Promise.all([
-        rs.getReader().closed,
-        ws.getWriter().closed.catch(e => {
-          assert_equals(e, error, 'the writable should be errored with the same object');
+for (const reason of [null, undefined, error1]) {
+  promise_test(() => {
```

Comment:
I kept the tests using Promise chains to stay consistent because all of the other tests in the file also used them. Although I agree using async/await is slightly more readable, would it be weird if some tests use async/await, while the rest still have Promise chains? (just wondering because I'm not sure what the norm is)

Reply:
Nevermind, I've made it an async function now, but if you think I should change it back, please let me know.

Reply:
> would it be weird if some tests use async/await, while the rest still have Promise chains?

I don't think that's a problem. @ricea do you have any opinions?

Reply:
I actually just talked to Adam offline, and he mentioned preferring async/await in newer tests now.